types/office-js/index.d.ts

```diff
@@ -24536,7 +24589,7 @@ declare namespace Excel {
         errorSubType?: BlockedErrorCellValueSubType | "Unknown" | "DataTypeRestrictedDomain" | "DataTypePrivacySetting" | "DataTypeUnsupportedApp" | "ExternalLinksGeneric" | "RichDataLinkDisabled" | "SignInError" | "NoLicense";
     }
     /**
-     * Represents the value of a cell containing a boolean.
+     * Represents the value of a cell containing a Boolean.
```

Comment:
is this change already in office-js-preview?

Reply:
No, just added it